modules/react/side-panel/stories/stories.tsx

```diff
@@ -174,14 +173,7 @@ export const Default = () => (
 
 const Template = props => (
   <div style={{height: '67vh', position: 'relative'}}>
-    <DeprecatedHeader brandUrl="#">
-      <Avatar
-        onClick={() => {
-          alert('clicked avatar');
-        }}
-      />
-      <PrimaryButton>Sign Up</PrimaryButton>
-    </DeprecatedHeader>
+    <Basic />
```

Comment:
Do we need a header here? Could we make a one? My worry is if we move this example to the MDX docs pattern, the import path will prevent copy/paste from working in codesandbox.